pandas/core/strings.py

```diff
@@ -646,9 +646,13 @@ def str_split(arr, pat=None, n=None, return_type='series'):
     """
     from pandas.core.series import Series
     from pandas.core.frame import DataFrame
+    from pandas.core.index import Index
 
     if return_type not in ('series', 'frame'):
```

Comment:
no I meant index just as an alias for series
it would return which type was executed (as u have it noe)